api/tests/organizational-entities/unit/application/api/organization-features-api_test.js

```diff
@@ -0,0 +1,25 @@
+import * as organizationEntitiesApi from '../../../../../src/organizational-entities/application/api/organization-features-api.js';
+import { OrganizationFeatureItem } from '../../../../../src/organizational-entities/domain/models/OrganizationFeatureItem.js';
+import { usecases } from '../../../../../src/organizational-entities/domain/usecases/index.js';
+import { expect, sinon } from '../../../../test-helper.js';
+
+describe('Unit | API | organization-features-api', function () {
+  describe('#getAllFeaturesFromOrganization', function () {
+    it('should return target profiles from organization', async function () {
+      const organizationId = Symbol('organizationId');
+
+      const getOrganizationFeatures = sinon.stub(usecases, 'getOrganizationFeatures');
+      getOrganizationFeatures
+        .withArgs({
+          organizationId,
+        })
+        .resolves([{}]);
+
+      // when
+      const result = await organizationEntitiesApi.getAllFeaturesFromOrganization(organizationId);
+
+      // then
+      expect(result[0]).not.to.be.instanceOf(OrganizationFeatureItem);
```

Comment:
**Question** : pourquoi `.not.to.be.instanceOf` et pas `.to.be.instanceOf`

Reply:
l'idée c'est de valider que l'on appel pas l'instance de votre Bounded Context .